src/Blocks/BlockButton.h

```diff
@@ -67,7 +67,7 @@ class cBlockButtonHandler :
 			case BLOCK_FACE_XM: return 0x2;
 			case BLOCK_FACE_XP: return 0x1;
 			case BLOCK_FACE_YM: return 0x0;
-			default:
+			case BLOCK_FACE_NONE:
```

Comment:
> adding an element to the enum without updating it a compile error.

By updating it, would you mean adding a condition for that new entry in every location where the enumeration is switch'd?